src/sentry/static/sentry/app/components/assistant/helper.jsx

```diff
@@ -61,32 +61,30 @@ const AssistantHelper = createReactClass({
   },
 
   render() {
-    const cueText = this.state.currentGuide
-      ? this.state.currentGuide.cue
-      : t('Need Help?');
     // isDrawerOpen and currentGuide/currentStep live in different places and are updated
     // non-atomically. So we need to guard against the inconsistent state of the drawer
     // being open and a guide being present, but currentStep not updated yet.
     // If this gets too complicated, it would be better to move isDrawerOpen into
     // GuideStore so we can update the state atomically in onGuideStateChange.
     let showDrawer = false;
-    if (
-      this.state.isDrawerOpen &&
-      (!this.state.currentGuide || this.state.currentStep > 0)
-    ) {
+    let {currentGuide, currentStep, isDrawerOpen} = this.state;
+    let isGuideCued = currentGuide;
```

Comment:
Since `isGuideCued` is a boolean, let's make that explicit with `currentGuide is not null` otherwise it will be the current guide.